### PR TITLE
cc.naver.com - Tracker

### PR DESCRIPTION
**Pull request by piquark6046**
# Prerequisites

### To avoid invalid pull requests, please check and confirm following terms

- [X] This is not an ad/bug report;
- [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [X] I have performed a self-review of my own changes;
- [X] My changes do not break web sites, apps and files structure.

# What problem does the pull request fix?

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [X] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [X] Filters maintenance.

# What issue is being fixed?

## Enter the issue address

- https://novel.naver.com/webnovel/weekday
- https://m.webtoons.com/en/
- https://m.webtoons.com/episodeList?titleNo=1798

#### m.webtoons.com

1. Visit [here](https://m.webtoons.com/en/).
2. Click one of highlighted elements:

<details><summary>Screenshot</summary>

Screenshot from 2022-06-18 08-50-05

</details>

3. Check the network log.

<details><summary>Screenshot</summary>

Screenshot from 2022-06-18 08-46-53

</details>

#### novel.naver.com

1. Visit [here](https://novel.naver.com/webnovel/weekday).
2. Click one of highlighted elements:

<details><summary>Screenshot</summary>

Screenshot from 2022-06-18 08-50-22

</details>

3. Check the network log.

<details><summary>Screenshot</summary>

Screenshot from 2022-06-18 08-45-11

</details>

### Related PRs and Issue

- https://github.com/AdguardTeam/AdguardFilters/issues/69674
- https://github.com/AdguardTeam/AdguardFilters/pull/97015
- https://github.com/AdguardTeam/AdguardFilters/pull/119488

## Terms

- [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: SpywareFilter/sections/specific.txt [hunk 1]
```diff
@@ -53,6 +50,8 @@
 /analytics.js$domain=freexcafe.com|rusvideos.tv|nakubani.ru|lun.com|songsara.net|newsmax.com|bolgehastanesi.com|ewtc.de|newsdig.tbs.co.jp
 ||assets.adobedtm.com^$domain=mcclatchydc.com|elnuevoherald.com|bellinghamherald.com|sacbee.com|charlotteobserver.com|newsobserver.com|miamiherald.com|ef.ru|algida.com.tr|bmw-motorrad.co.uk|cif.com.tr|clearhaircare.com|domestos.com|dove.com|elidor.com.tr|f1sport.autorevue.cz|geico.com|gittigidiyor.com|hellmanns.com.tr|herkessofraya.com|johnsonsbaby.com.tr|lipton.com|lnk.to|lux.com|magnumicecream.com|neutrogena.com.tr|omo.com|rexona.com|samsung.com|sap.com|schwarzkopf.com.tr|signalturkiye.com|msnbc.com|nbcnews.com|kbb.com|eurosport.ru|timgate.it|intermiles.com|oxfordmail.co.uk
 !
+||cc.naver.com/cc|$image,other,ping
+/^https:\/\/cc\.(loginfra|naver)\.com\/cc\?a=/$image
```
Comment (Alex-302) [inline SpywareFilter/sections/specific.txt:54]: Not need to add generic regexp only for two urls. Just create new rules, similar to `||cc.naver.com/cc|$image,other,ping`

Comment (piquark6046) [reply SpywareFilter/sections/specific.txt]: Okay.

https://github.com/AdguardTeam/AdguardFilters/pull/122182/commits/406cde88caa32b208cb2071efa520718f56d0654

**Review by zloyden — APPROVED** [review]

File: SpywareFilter/sections/specific.txt [hunk 2]
```diff
@@ -53,6 +50,9 @@
 /analytics.js$domain=freexcafe.com|rusvideos.tv|nakubani.ru|lun.com|songsara.net|newsmax.com|bolgehastanesi.com|ewtc.de|newsdig.tbs.co.jp
 ||assets.adobedtm.com^$domain=mcclatchydc.com|elnuevoherald.com|bellinghamherald.com|sacbee.com|charlotteobserver.com|newsobserver.com|miamiherald.com|ef.ru|algida.com.tr|bmw-motorrad.co.uk|cif.com.tr|clearhaircare.com|domestos.com|dove.com|elidor.com.tr|f1sport.autorevue.cz|geico.com|gittigidiyor.com|hellmanns.com.tr|herkessofraya.com|johnsonsbaby.com.tr|lipton.com|lnk.to|lux.com|magnumicecream.com|neutrogena.com.tr|omo.com|rexona.com|samsung.com|sap.com|schwarzkopf.com.tr|signalturkiye.com|msnbc.com|nbcnews.com|kbb.com|eurosport.ru|timgate.it|intermiles.com|oxfordmail.co.uk
 !
+||cc.loginfra.com/cc?a=$image
+||cc.naver.com/cc?a=$image
+||cc.naver.com/cc|$image,other,ping
```
Comment (zloyden) [inline SpywareFilter/sections/specific.txt:55]: What if you create one rule?
```
||cc.naver.com/cc$image,other,ping
```

Comment (piquark6046) [reply SpywareFilter/sections/specific.txt]: No problem.

**Review by zloyden — APPROVED** [review]

**Pull request merged**